osf-model/src/main/resources/ontology/osf-model.rdf

```diff
@@ -773,6 +773,65 @@
         <rdfs:range rdf:resource="&xsd;string"/>
     </owl:DatatypeProperty>
 
+    <owl:DatatypeProperty rdf:about="&osfbom;hasContent>
+        <rdfs:label xml:lang="en">hasContent</rdfs:label>
+        <rdfs:comment xml:lang="en">The text of this comment</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;string"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <!-- "hasNode" is already taken -->
```

Comment:
What if we just widened the semantics of the existing `hasNode` to be a more generic relationship between a non-`nodes` JSON-API type (like `files` or `comments`) and a `nodes`?

E.g. change the domain of `osf:hasNode` to be an `osf:OSFBusinessObject`?